tests/medium/__init__.py

```diff
@@ -177,3 +180,14 @@ def create_file(self, path, content):
         command = self.command_as_list(["mkdir", "-p", dir_path, path])
         if not self._exec_command(command)[0]:
             raise BaseException("Couldn't create {} in container".format(path))
+
+    @nottest
```

Comment:
It has test on a word boundary which appears to be good enough for nose. I was surprised too:

http://nose.readthedocs.org/en/latest/writing_tests.html